### PR TITLE
Updated arc diagram: Grouped the models by model type

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/BeSLighthouse/issues/576

Models are grouped by type: 
1. Classic
2. LLM
3. Not Tracked (not tracked under Be-Secure)

Screenshot from 2024-02-16 15-30-29

### CODE REVIEW

File: src/pages/ModelOfInterest/ArcDiagram/index.tsx [hunk 1]
```diff
@@ -202,7 +217,27 @@ const ArcDiagram: React.FC = () => {
...
+      </div>
+
+      <div className="container" style={{ display: "flex", alignItems: "center", marginBottom: "5px", marginTop: "2px" }}>
+        <div className="circle dependency" style={{ backgroundColor: "#EC5800", width: "10px", height: "10px", borderRadius: "50%", marginRight: "5px", marginLeft: "20px"}}></div>
+        <p style={{ fontSize: "13px" }}>LLM (Model)</p>
+      </div>
+
+      <div className="container" style={{ display: "flex", alignItems: "center", marginBottom: "5px", marginTop: "2px" }}>
+        <div className="circle dependency" style={{ backgroundColor: "blue", width: "10px", height: "10px", borderRadius: "50%", marginRight: "5px", marginLeft: "20px"}}></div>
+        <p style={{ fontSize: "13px" }}>Unknown</p>
```
Comment (harimohanr) [inline src/pages/ModelOfInterest/ArcDiagram/index.tsx:234]: @pramit-d We wouldnt be having an Unknown type model. Its either Classic or LLM as of now.

**Review by harimohanr — CHANGES_REQUESTED** [review]
@pramit-d Changes suggested

File: src/pages/ModelOfInterest/ArcDiagram/index.tsx (hunk 1 above)
Comment (pramit-d) [reply src/pages/ModelOfInterest/ArcDiagram/index.tsx]: @harimohanr updated.

File: src/pages/ModelOfInterest/ArcDiagram/index.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as d3 from 'd3';
 
 const ArcDiagram: React.FC = () => {
   useEffect(() => {
-    const githubRawUrl = 'https://raw.githubusercontent.com/Be-Secure/besecure-assets-store/main/models/model-metadata.json';
+    const githubRawUrl = 'https://raw.githubusercontent.com/pramit-d/besecure-assets-store/main/models/model-metadata.json';
```
Comment (sudhirverma) [inline src/pages/ModelOfInterest/ArcDiagram/index.tsx:6]: @pramit-d Please Be-Secure namespace

Comment (sudhirverma) [reply src/pages/ModelOfInterest/ArcDiagram/index.tsx:6]: and move the URL to datastore.ts file

Comment (pramit-d) [reply src/pages/ModelOfInterest/ArcDiagram/index.tsx]: @sudhirverma done.

**Review by sudhirverma — APPROVED** [review]

**Pull request merged**